conductr_cli/bndl_utils.py

```diff
@@ -35,14 +35,8 @@ def detect_format_dir(dir):
             os.path.isfile(os.path.join(dir, 'repositories')) and \
             os.path.isfile(os.path.join(dir, 'manifest.json')):
         return 'docker'
-    elif \
-            os.path.isfile(os.path.join(dir, 'runtime-config.sh')):
-        return 'bundle'
-    elif \
-            os.path.isfile(os.path.join(dir, 'bundle.conf')):
-        return 'bundle'
     else:
-        return None
+        return 'bundle'
```

Comment:
Prior to this PR, the format argument identified only the format of the input. Now the format argument identifies the input and output format. Let's say we have an application directory as an input that does contain a `bundle.conf` or `runtime-config.sh`. The old auto-detection logic would return `None` and therefore the `bndl` command would abort saying that the `--format` needs to be specified. However,  we want to create a `bundle` for this input type. We also don't want that the user needs to specify the `--format bundle` argument explicitly given this is a common use case. The user only needs to specify the `--format configuration` argument explicitly because we cannot distinguish between bundle and configuration in the auto-detection function.

Also, if the input is an archive, the auto-detection function will set the format to `bundle` as well. It does not check for a `bundle.conf` or `runtime-config.sh` file. This logic has not been changed. So with this PR, we are also consistent across input directories and archives.

@longshorej Do you have any observations in this change, i.e. do we run into other issues with this change that I am not aware of?